core/src/main/java/org/apache/calcite/util/BuiltInMethod.java

```diff
@@ -629,6 +629,8 @@ public enum BuiltInMethod {
   SUBMULTISET_OF(SqlFunctions.class, "submultisetOf", Collection.class,
       Collection.class),
   ARRAY_DISTINCT(SqlFunctions.class, "distinct", List.class),
+  ARRAY_MAX(SqlFunctions.class, "max",  List.class),
```

Comment:
i does that because i refer to array_reverse implmented by snuyanzin, the function also names reverse. so i do it too @JiajunBernoulli do you think it is need? i can modify all